adaptivefiltering/dataset.py

```diff
@@ -136,11 +131,9 @@ def show_interactive(self):
 
     def save(self, filename, compress=False, overwrite=False):
         """Store the dataset as a new LAS/LAZ file
-
```

Comment:
I like most of this whitespace for readability. Did pre-commit remove that or did you do it on purpose?

Reply:
they were removed automatically